src/components/userProfile/UserOverview.js

```diff
@@ -150,38 +151,15 @@ class UserOverview extends Component {
   }
 
   renderPersonality() {
-    const personalityArray = this.state.personality;
-    const scoreArray = this.state.score;
+    const tagList = [];
+    this.state.personality.map((item) => {
+      tagList.push({ title: item.option, point: item.score });
+    });
 
-    const fontStyle = [
-      {
-        color: '#cdd2d2',
-        fontSize: dimensions.fontWeight * 12,
-      },
-      {
-        color: '#757b7c',
-        fontSize: dimensions.fontWeight * 16,
-      },
-      {
-        color: '#2e3031',
-        fontSize: dimensions.fontWeight * 20,
-      },
-    ];
     return (
       <View style={styles.sectionContainer}>
         <Text style={styles.sectionName}>Personality</Text>
-        <View style={{ flexDirection: 'row', flexWrap: 'wrap' }}>
-          {
-            personalityArray.map((value, index) =>
-              (
-                <Text
-                  key={index}
-                  style={[fontStyle[scoreArray[index]], styles.personality]}>
-                  {value}
-                </Text>
-              ))
-          }
-        </View>
+        <CloudTag tagList={tagList} width={250} />
```

Comment:
Yes! Total width is fixed.
If this size is not reasonable, it is available to change this size.